src/System.Private.CoreLib/src/System/Runtime/CompilerServices/ClassConstructorRunner.NonPortable.cs

```diff
@@ -31,10 +31,10 @@ private static int CurrentManagedThreadId
         {
             get
             {
-                return ManagedThreadId.Current;
+                return Environment.CurrentManagedThreadId;
             }
         }
 
-        private const int ManagedThreadIdNone = ManagedThreadId.ManagedThreadIdNone;
+        private const int ManagedThreadIdNone =0;
```

Comment:
Nit: there should be extra space between 0 and `ManagedThreadId.ManagedThreadIdNone`